source/features/reactions-avatars.css

```diff
@@ -25,10 +29,14 @@ button.reaction-summary-item { /* `button` excludes the "Add reaction" icon */
 	margin-left: -0.5em;
 	vertical-align: middle;
 	background: #efefef; /* Placeholder before the images load */
```

Comment:
Can you use an existing variable here instead?

Reply:
There's no variable in https://primer.style/primitives/ that corresponds with `#efefef` for light mode. This is using `--color-bg-canvas-inset`:
![image](https://user-images.githubusercontent.com/1441704/102628746-7a4a3100-414a-11eb-800c-c1c23c2cd636.png)
![image](https://user-images.githubusercontent.com/1441704/102628752-7b7b5e00-414a-11eb-9e26-95105deb97b2.png)